src/operations/create_collection.ts

```diff
@@ -137,7 +137,10 @@ export class CreateCollectionOperation extends CommandOperation<Collection> {
 
       if (encryptedFields) {
         // Creating a QE collection required min server of 7.0.0
-        if (server.description.maxWireVersion < MIN_SUPPORTED_QE_WIRE_VERSION) {
+        if (
+          !server.loadBalanced &&
```

Comment:
Given the conversation in drivers, would it be better to keep the logic as is and temporarily skip the failing test? Or, alternatively, keep this implementation and file a follow up to refactor the load balancer logic to correctly check the connection wire version? Either way, some sort of follow up ticket is likely good to create and note in this PR

Reply:
I've created https://jira.mongodb.org/browse/NODE-5353 and updated the comment here.